README.md

```diff
@@ -6,6 +6,15 @@
 
 Python client for the [kubernetes](http://kubernetes.io/) API.
 
+## Requirements
+
+A version of python with OpenSSL version 1.0.0 or later.
```

Comment:
Ah ok, should I make this a macOS-specific instruction then? If so, I'd mention using brew as well.